### ISSUE TITLE
Cannot boot to BigSur installer - Legion Y540

### ISSUE THREAD

**Issue by HarryHeres** [xiaoMGitHub/LEGION_Y7000Series_Hackintosh#184]
Hi! I had Catalina 10.15.X with Clover r5107 working just fine, so I went to upgrade to Big Sur. Unfortunately, still haven't found a solution to why I cannot boot into the Mac installation. I have successfully installed Clover r5130 to my flash drive alongside the installation package itself, can boot to clover and choose booting platform (Catalina/Windows/BigSur installation). However, no matter which I choose (excluding Windows working just fine), I cannot seem to boot. I have managed to take a photo of my error, including my specs below: 

Legion Y540-15IHR 2019
- I7 9750H 
- Nvidia 1660Ti (disabled)
- Samsung 970 M.2 NVMe EVO 500GB for system partitions

I have also tried to run your script for the Bios modification, even though my Bios settings worked just fine with Catalina, but I haven't been to run it. 

![166332625_480509792991407_5387800913662373107_n](https://user-images.githubusercontent.com/74781187/112885847-82aef200-90d1-11eb-88ae-67036d0d1cb2.jpg)

![image](https://user-images.githubusercontent.com/74781187/112886221-eb966a00-90d1-11eb-819d-20e67b42d5e6.png)

Any help appreciated!

**Comment by xiaoMGitHub** [comment]
It seems that the Chinese script has garbled characters in your environment. First, you need to turn off the real-time scanning of the anti-virus software, and then re-extract my script.

**Comment by HarryHeres** [comment]
I have managed to run previous macOS version (Catalina 10.15), so I'll be sticking with hit for now and will try to update sometime later (around summer).

**Comment by hewad23** [comment]
I have a similar model as yours and currently running Big Sur latest version. I made some changes to the old OC because this release didn't work for me either.

**Comment by HarryHeres** [comment]
May I know what changes in particular did you make? 😄

**Comment by hewad23** [comment]
> May I know what changes in particular did you make? 😄

I can share the efi and check if that works for you.

**Comment by HarryHeres** [comment]
That would be awesome!

**Comment by maxime843300** [comment]
hi @hewad23 can you share it with me i can't install big on either

**Comment by SaravananSudesh** [comment]
I am also facing the same issue. Catalina worked but Big Sur isn’t working on my Legion Y540
-My Specs-
i7-9760H
32 GB Ram
Intel UHD 630 (Nvidia GTX 1650 disabled)
1TB WD M.2

**Comment by HarryHeres** [comment]
> 
> 
> > May I know what changes in particular did you make? 😄
> 
> I can share the efi and check if that works for you.

Hi, would it be possible for you to share the EFI? 😊

**Comment by HarryHeres** [comment]
So I have managed to install the Big Sur update on my Legion Y540. I have disabled all of the security features of Windows (Defender, Eset, ...). Then, I ran all the .exe files in the Bios Modifier folder, which generated some info about the current BIOS configuration, etc. After that, I ran the modifying script itself and despite the unreadable characters, it executed successfully (option 2 -> 4 -> 5 in order). I copied @xiaoMGitHub's EFI folder alongside Big Sur installation package to my flash disk and booted it. The installation crashed almost everytime on "12 minutes remaining", which caused restarts to it. I simply let it crash and chose a new bootable partition, which showed up in Clover. It finished the installation proccess and went black for approx. 5 minutes (probably a monitor backlight not properly configured). Then it lightened up with the macOS configuration process and successfully booted into Big Sur! 

I'm closing this issue with this comment, shall anyone run into similar problems like I did.
